### DISCUSSION TITLE
Show calendar or calendar entries on NSPanel

### DISCUSSION THREAD

**Discussion by Belgadon** [Blackymas/NSPanel_HA_Blueprint#2371] [Ideas]
_Edit_: Just found https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/2318 

Hi all,

maybe it is already possible.

I think it would be great to have NSPanel show a calender, or the next calendar entries.

Space wise I dont really know what would look good on the rather small screen.
Maybe only text withouf grafiks

- Whole month
- Two weeks
- One day
- Next n entries
- ...

Examples:
![image](https://github.com/user-attachments/assets/752d80aa-a0b0-4b86-a4ac-eec92a1b9a88)
![image](https://github.com/user-attachments/assets/9aab6c07-e14a-4a0d-be0f-eb4068ee5b8f)
![image](https://github.com/user-attachments/assets/500c5601-e8ed-4392-b216-38bf7e57c0d8)
![image](https://github.com/user-attachments/assets/827a8aba-81f8-4c95-804e-64e821e48414)

**Comment by petersuba** [comment]
Isn't this covered by the suggested feature #1858 ?

**Comment by pkleingu** [comment]
this would be nice, in combination with https://www.creatingsmarthome.com/index.php/2023/04/12/home-assistant-advanced-nord-pool-cheapest-hours-with-local-calendar-support/#google_vignette you can see when your car is charging or diswascher is running on date and time.
